stderr of test 'wlr50` in directory 'sql/test/wlcr` itself:


# 20:24:40 >  
# 20:24:40 >  "/usr/bin/python2" "wlr50.py" "wlr50"
# 20:24:40 >  

# builtin opt 	gdk_dbpath = /export/scratch1/mk/wlcr//Linux/var/monetdb5/dbfarm/demo
# builtin opt 	gdk_debug = 0
# builtin opt 	gdk_vmtrim = no
# builtin opt 	monet_prompt = >
# builtin opt 	monet_daemon = no
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_port = 50429
# cmdline opt 	mapi_usock = /var/tmp/mtest-13781/.s.monetdb.50429
# cmdline opt 	gdk_dbpath = /export/scratch1/mk/branches/wlcr//Linux/var/MonetDB/mTests_sql_test_wlcrclone
#client1:!ERROR:MALException:wlr.getMaster:Could not get read access to 'demo'config file
#client1:!ERROR:MALException:sql.replicate:Replicator not started, call setmaster() first 
MAPI  = (monetdb) /var/tmp/mtest-13781/.s.monetdb.50429
QUERY = call wlr.master('demo');   
ERROR = !Could not get read access to 'demo'config file
MAPI  = (monetdb) /var/tmp/mtest-13781/.s.monetdb.50429
QUERY = call wlr.replicate();
ERROR = !Replicator not started, call wlr.master() first 

# 20:24:40 >  
# 20:24:40 >  "Done."
# 20:24:40 >  

